persistence/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/persistence/PersistenceComponents.scala

```diff
@@ -31,9 +31,11 @@ trait ReadSidePersistenceComponents extends WriteSidePersistenceComponents {
   def actorSystem: ActorSystem
   def executionContext: ExecutionContext
   def materializer: Materializer
-  def configuration: Configuration
 
-  lazy val readSideConfig: ReadSideConfig = ReadSideConfig(configuration)
+  def conf: Config = configuration.underlying
+  protected def configuration: Configuration
```

Comment:
Why `protected`? Technically, this is a public API, so reducing visibility could possibly break something (though I think it's unlikely). I think it's better to keep it public, but `@deprecated`.